File: benchmark/results/java-invocation-inline/run-001/b-baseline/output/com/metaengine/demo/identity/services/AuthService.java
package com.metaengine.demo.identity.services;

import com.metaengine.demo.identity.aggregates.User;
import java.util.List;

/** Domain service handling authentication for users. */
public class AuthService {
    /** Authenticate and create a user session from the provided partial input. */
    public User create(User input) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Find an authenticated user by id. Returns null when not found. */
    public User findById(String id) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** List up to {@code limit} authenticated users. */
    public List<User> list(int limit) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Sign out / revoke a user by id. */
    public void delete(String id) {
        throw new UnsupportedOperationException("not implemented");
    }
}

File: benchmark/results/python-invocation-script/run-004/a-mcp/output/identity/services/auth_service.py
from identity.aggregates.user import User

"""AuthService service."""
class AuthService:
    def create(self, input: dict) -> User:
        raise NotImplementedError('not implemented')
    def find_by_id(self, id: str) -> User | None:
        raise NotImplementedError('not implemented')
    def list(self, limit: int) -> list[User]:
        raise NotImplementedError('not implemented')
    def delete(self, id: str) -> None:
        raise NotImplementedError('not implemented')

